xds/src/main/java/io/grpc/xds/CdsLoadBalancer2.java

```diff
@@ -188,7 +188,8 @@ private void handleClusterDiscovered() {
       if (root.result.lbPolicy() == LbPolicy.RING_HASH) {
         lbProvider = lbRegistry.getProvider("ring_hash");
         lbConfig = new RingHashConfig(root.result.minRingSize(), root.result.maxRingSize());
-      } else {
+      }
+      if (lbProvider == null) {
```

Comment:
Should we set `lbConfig = null`? Just for safety?

Reply:
Oops, just saw you comment.